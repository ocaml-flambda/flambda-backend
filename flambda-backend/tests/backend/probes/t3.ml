let () =
  [%probe "name" (Printf.printf "default\n")];
  [%probe "name" ~enabled_at_init:true (Printf.printf "enabled at init\n")];
  [%probe "name" ~enabled_at_init:false (Printf.printf "not enabled at init\n")];
  ()
