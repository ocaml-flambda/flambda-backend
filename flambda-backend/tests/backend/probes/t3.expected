File "t3.ml", line 1:
Error: Error producing assembly code for t3.ml: Inconsistent use of ~enabled_at_init in [%probe name ..] at t3.ml:3,2--75
